### ISSUE TITLE
Can't send single mails with stored templates (Postmark) @bug

### ISSUE THREAD

**Issue by Tobeyforce** [anymail/django-anymail#223]
Using anymail 8.1 with python 3.8.2 it seems like I cannot send a single message using a stored template.
I am forced to use merge_data to get it to work, which isn't a major problem but I thought sending to one user allowed usage of stored templates for postmark?

E.g, this will fail:
```
message = AnymailMessage(
        from_email="myfromemail@domain.com",
        to=["example@gmail.com"],
        template_id="my template alias"})

message.send()
```

With this response:
```
Postmark API response 422 (Unprocessable Entity):
{
  "ErrorCode": 1120,
  "Message": "A 'TemplateModel' is required when using a template to send an email."       
}

```

However this works:

```
message = AnymailMessage(
        from_email="myfromemail@domain.com",
        to=["example@gmail.com"],
        template_id="my template alias"},
        merge_data={ 
            'example@gmail.com': { 'mymergedata': 'somedata'},
        })
```

Am I missing something? Tried using EmailMessage as well but no success. Or is merge_data enforced regardless if you want to include a context or not?

**Comment by medmunds** [comment]
Yes, I think this is a bug in Anymail's Postmark backend. Anymail should probably force a TemplateModel into the payload whenever TemplateAlias or TemplateId are there.

Until then, you should be able to use an empty merge_global_data as a workaround:
```python
message = AnymailMessage(
    from_email="myfromemail@example.net",
    to=["recipient@example.com"],
    template_id="my template alias",
    merge_global_data={},
)
```

**Comment by Tobeyforce** [comment]
This seems to work for now, thanks. Loving anymail, my only two pain points currently are:
1. Would be awesome to have a tutorial / guide how to handle exceptions & logging with anymail, e.g using Celery. Django-mailer had some cool logging abilities but unfortunately didn't blend great with anymail when the esp-specific features were needed.
2. Being able to set a backup email backend for admin messages in settings.py, instead of passing the backup-settings to the get_connection everytime.
Any plans for this?

**Comment by medmunds** [comment]
@Tobeyforce please open separate [discussions](https://github.com/anymail/django-anymail/discussions) for each other topic. (Curious what problems you had with django-mailer, but this Postmark bug isn't the right place to discuss it.)

**Comment by medmunds** [comment]
Fix released in v8.3
